src/orquestra/sdk/_base/_driver/_exceptions.py

```diff
@@ -149,14 +149,26 @@ def __init__(self, workflow_run_id: str):
         super().__init__(workflow_run_id)
 
 
+class TaskRunLogsNotFound(Exception):
+    """
+    Raised when a Task Run's Logs cannot be found
+    """
+
+    def __init__(self, workflow_run_id: str, task_invocation_id: str):
```

Comment:
**non-blocking,question**: is there a reason we're not using `WorkflowRunId` and `TaskInvocationId` as type annotations in this module?